configure.ac

```diff
@@ -36,10 +36,10 @@ AM_SILENT_RULES([yes])
 # 3. if interfaces were removed, then use C+1:0:0
 ##########################################################################
 
-ODP_LIBSO_VERSION=115:0:2
+ODP_LIBSO_VERSION=116:0:3
```

Comment:
As we are not providing ABI-level compatibility between releases (and we are not ATM), this always should be XXX:0:0.
Especially since we removed `odp_packet_unshare_len()`. So proper version should be 116:0:0

Reply:
@lumag what is ATM stands for?

Reply:
@muvarov "at this moment"

Reply:
@lumag  I like that idea to keep LIBSO version always the same as API version. But when I said that first time than Anders said that it's not correct and we have to follow the same rules as libcurl. So for some last releases we followed that magic scheme. Which might be a not good fit for mainline version.